automation/run-tests.sh

```diff
@@ -14,6 +14,9 @@ fi
 NET0="nmstate-net0"
 NET1="nmstate-net1"
 
+#Valid TEST_TYPE are: all, tox_code_style, tox_py27, tox_py36, integ
+: ${TEST_TYPE:="integ"}
```

Comment:
@EdDev would you be fine with running all tests by default but allowing you to run integration tests only by setting this variable?

Reply:
All would be both unit and integration tests.